tools/DSharpPlus.Tools.CodeBlockLanguageListGen/FromCodeAttribute.LanguageList.template

```diff
@@ -0,0 +1,16 @@
+// <auto-generated/>
+// Last modified on {{Date}}
+
+using System;
+using System.Collections.Frozen;
+using System.Collections.Generic;
+
+namespace DSharpPlus.Commands.ContextChecks;
+
+public sealed partial class FromCodeAttribute : Attribute
```

Comment:
just `partial class` so the other modifies can be changed in code

Reply:
No modifications can be made to the behavior as it's the argument converter that's doing the work. Remember how you didn't want context checks to be able to mutate parameter values?

Reply:
the modifiers, not the behaviour

Reply:
say, you want to remove sealed for whatever reason. that now needs to be changed in two places and the tool needs to re-run